apps/src/lib/levelbuilder/lesson-editor/UploadImageForm.jsx

```diff
@@ -50,9 +55,17 @@ export default function UploadImageForm() {
       });
   };
 
+  const saveImagesToS3 = () => {
+    for (const image of formDataForImages.entries()) {
```

Comment:
Mentioned above, but since we're just looping over the `formDataForImages` to create the `new FormData()` we actually submit below, perhaps instead of `formDataForImages` it should be `listOfImageFiles` or such. I don't hate this way, but it does slightly obscure what's happening.